isis/src/base/apps/skymap/main.cpp

```diff
@@ -247,6 +252,13 @@ bool sky2map::Xform(double &inSample, double &inLine,
   if(p_incam->Sample() > p_inputSamples + 0.5) return false;
   if(p_incam->Line() > p_inputLines + 0.5) return false;
 
+  p_incam->SetImage(p_incam->Sample(), p_incam->Line());
+  
+  // Check that the new line and sample projects close to the input Right Ascension 
+  // and Declination
+  if (abs(p_incam->RightAscension() - lon) > 0.1) return false;
+  if (abs(p_incam->Declination() - lat) > 0.1) return false;
```

Comment:
@Kelvinrr These lines were added to avoid dual projection. Basically any look vector projects into two points in right ascension and declination, so if you project to 0, 0 you also project onto 0, 180. This checks avoids look vectors projecting into the inverse right ascension, declination

Reply:
Did you update the test data? We just got to make sure we aren't adding another failing test if we dont have to.

Reply:
I have updated data, we usually check it in once the PR is merged

Reply:
kk just making sure